src/engine-actions/translate-swap.js

```diff
@@ -63,7 +63,7 @@ async function translateSwap (takerAddress, swapHash, amount, maxTime, finalCltv
     amt: amount,
     finalCltvDelta,
     cltvLimit,
-    feeLimit: DEFAULT_FEE_LIMIT
+    feeLimit: DEFAULT_FEE_LIMIT.toString()
```

Comment:
This seems to be now a string, so is `toString()` does not seem necessary. Also, same comment as the prior variable that was changed to a string (just ensure that it is truly what you wanted)